sycl/doc/design/CompileTimeProperties.md

```diff
@@ -806,26 +814,57 @@ into one (or both) of the following:
 In both cases, the decoration is a single **UserSemantic** decoration where the
 string literal is the same as the string literal in the LLVM annotation.
 
-When a SYCL structure member property needs to be represented in SPIR-V,
-however, we prefer to represent each property as an extended SPIR-V decoration
-rather than using a **UserSemantic** decoration.  There is no existing
-mechanism in the SPIR-V LLVM Translator to generate extended decorations like
-this, so we propose the following new mechanism.
+An exception to this is for a selection of FPGA-related decorations. If these
+are supported during translation from LLVM IR to SPIR-V the corresponding
+decorations will be generated, and otherwise it will fall back to creating a
+single **UserSemantic** decoration. In general these decorations occur in the
+annotation string as a series of **{X}** and **{X:Y}** where **X** is a reserved
+name and **Y** is one or more words and numbers separated by **,** or **:**,
+depending on the decoration.
+
+As such we propose an extension to this functionality with the following
+changes:
+
+* To bring it in line with the format of the metadata decorations, the parsing
+  of these decorations should allow the use of SPIR-V decoration identifiers
+  rather than reserved names. With this there need not be any agreement between
+  the translator and LLVM IR producer, as the identifiers are specified by the
+  SPIR-V specification.
+* For decorations parsed with decoration identifiers, only the **,** delimiter
```

Comment:
I was reading the HTML version of this design doc, and I noticed that the formatting of `**,**` does not work well, and the resulting text is very confusing.  I'd suggest changing this to something like:

> only the coma delimiter (`,`) is valid ...

Reply:
I see what you mean. I changed it in the mention of delimiters above and excluded the symbol mention in this part.